hibernate-core/src/main/java/org/hibernate/sql/ast/tree/cte/package-info.java

```diff
@@ -24,7 +24,7 @@
  * 			* select - `QuerySpec`
  * 			* delete - `DeleteStatement`
  * 			* update - `UpdateStatement`
- * 			* insert-select - `InsertSelectStatement
+ * 			* insert-select - `InsertStatement
```

Comment:
there is a missing `\`` at the end

Reply:
I'm not going to force-push to fix a missing quite in a comment that occurs in code that's not in any way related to this PR. Sorry.